exporter/wavefrontexporter/wavefront_test.go

```diff
@@ -0,0 +1,3 @@
+package wavefrontexporter
+
+// TODO: Add tests.
```

Comment:
If there're no tests at this moment then you need to put a `nocover` in this package, similar to https://github.com/census-instrumentation/opencensus-service/blob/master/internal/collector/jaeger/.nocover.

Reply:
From `check-test-files.sh`, `nocover` seems to work only when there are no `*_test.go` files.
I've still added one. But, the following exporters also have dummy test files without `nocover`.

```
./jaegerexporter/jaegerexporter_test.go
./kafkaexporter/kafkaexporter_test.go
./stackdriverexporter/stackdriverexporter_test.go
./datadogexporter/datadog_test.go
./honeycombexporter/honeycombexporter_test.go
./exporterwrapper/exporterwrapper_test.go
```